stack.yaml

```diff
@@ -1,4 +1,4 @@
-resolver: lts-13.2
+resolver: lts-13.3
```

Comment:
Oops. Didn't mean to change this..

Reply:
That's fine.  There's nothing wrong with supporting a newer Stack version